package.json

```diff
@@ -33,9 +33,11 @@
     "babel-polyfill": "^6.26.0",
     "bs58": "^4.0.1",
     "cross-fetch": "^2.1.1",
+    "ethereumjs-util": "^5.2.0",
```

Comment:
Do we still need this?

Reply:
@nick No I don't think it's needed. It's currently being used for attestation signature validation. https://github.com/OriginProtocol/origin-js/blob/d18dddb8f6c8348896d5851d8da5d6c750b79e31/src/resources/users.js#L189

We should be able to do all this with web3. Created an issue for it so I don't forget. https://github.com/OriginProtocol/origin-js/issues/153